assets/src/edit-story/components/panels/sizePosition.js

```diff
@@ -156,11 +180,51 @@ function SizePositionPanel({
     [lockAspectRatio]
   );
 
+  usePresubmitHandler(({ rotationAngle: newRotationAngle }) => {
+    return {
+      rotationAngle: setMinMax(newRotationAngle, MIN_MAX.ROTATION),
```

Comment:
This was good as it was before -- for example, it's now setting 361 to 359 even though it should set it to 1 (as it did before).

Reply:
Waiting for the final confirmation from Sam on this.